frontend/public/components/catalog/catalog-items.jsx

```diff
@@ -98,38 +98,74 @@ const pageDescription =
   'Catalog. Cluster admins can install additional apps which will show up here automatically.';
 
 // Filter property white list
-const filterGroups = ['kind'];
+const filterGroups = ['type'];
```

Comment:
Why are you changing `kind` -> `type`?

Reply:
filterGroup has now been changed to `type` instead of `kind`. The ability of a `type` is to hold more than one `kind` wherever required.

Reply:
Do you have a use case that fits this type of change? The current acceptance criteria for the story doesn't seem like it needs this change at the moment.